packages/gatsby/lib/bootstrap/index.js

```diff
@@ -77,6 +78,8 @@ const autoPathCreator = async (program: any) => {
 
 module.exports = async (program: any) => {
   console.log(`lib/bootstrap/index.js time since started:`, process.uptime())
+  // Fix program directory path for windows env
+  program.directory = slash(program.directory);
```

Comment:
Hmmm yeah, ok I see what you mean. Yeah this should be fine keeping this here because both build & develop call bootstrap. The one command that doesn't "new" doesn't use `program.directory`.